cartridges/int_styla/cartridge/scripts/StylaMain.js

```diff
@@ -131,247 +128,267 @@ function getContentVersion(config) {
 }
 
 
 /**
  * Find a matching magazine configuration.
- * 
+ *
  * @param {string} path Original URL before redirect.
  * @returns {Object} Magazine configuration if a matching magazine configuration was found, else returns null.
  */
 function getConfigForAlias(path) {
-	var result = null,
-		getContentVer = false, // skip request for content version, we only need SEO request to determine HTTP status code
-		magazineConfig;
-	
-	if (STYLA_ENABLED) {
-		magazineConfig = getMagazineConfiguration(path, getContentVer);
-		if (magazineConfig.valid) {
-			result = magazineConfig;
-		}
-	}
-	
-	return result;
+    var result = null,
+        getContentVer = false, // skip request for content version, we only need SEO request to determine HTTP status code
+        magazineConfig;
+
+    if (STYLA_ENABLED) {
+        magazineConfig = getMagazineConfiguration(path, getContentVer);
+        if (magazineConfig.valid) {
+            result = magazineConfig;
+        }
+    }
+
+    return result;
 }
 
 
-
 /**
- * Get the Styla content to be injected into the page's header and body.   
+ * Get the Styla content to be injected into the page's header and body.
  */
 function getRenderContent() {
-	var result = null,
-		magazineConfig,
-		seoResponse = null,
-		
-		magazinePath,
-		queryString,
-		getContentVer;
-	
-	if (STYLA_ENABLED) {
+    var result = null,
+        magazineConfig,
+        seoResponse = null,
+
+        magazinePath,
+        queryString,
+        getContentVer;
+
+    if (STYLA_ENABLED) {
 		// read parameters that were passed via remote-include
-		magazinePath  = request.httpParameterMap.magazinepath.submitted  && request.httpParameterMap.magazinepath.stringValue || null;
-		queryString   = request.httpParameterMap.querystring.submitted   && request.httpParameterMap.querystring.stringValue  || null;
-		getContentVer = request.httpParameterMap.getcontentver.submitted && request.httpParameterMap.getcontentver.stringValue === 'true';
-		
-		if (!empty(magazinePath) && magazinePath.indexOf('?') < 0 && !empty(queryString)) {
-			magazinePath += '?' + queryString;
-		}
-		
-		magazineConfig = getMagazineConfiguration(magazinePath, getContentVer);
-		if (magazineConfig['valid']) {
-			// found a magazine configuration that matches the original request's path 
-			result = {
-				MagazineConfiguration: magazineConfig,
-				SeoResponse: magazineConfig.seoResponse || null
-			};
-		}
-	}
-	
-	return result;
+        magazinePath = request.httpParameterMap.magazinepath.submitted && request.httpParameterMap.magazinepath.stringValue || null;
+        queryString = request.httpParameterMap.querystring.submitted && request.httpParameterMap.querystring.stringValue || null;
+        getContentVer = request.httpParameterMap.getcontentver.submitted && request.httpParameterMap.getcontentver.stringValue === 'true';
+
+        if (!empty(magazinePath) && magazinePath.indexOf('?') < 0 && !empty(queryString)) {
+            magazinePath += '?' + queryString;

```

Comment:
we should ignore / remove query parameters to since query parameters have no influence on the content or seo. So `/foobar` and `/foobar?foo=bar` should both yield to `/foobar` for the seo cache key and the seo api path parameter.